### ISSUE TITLE
[Accessibility] Some elements in the settings page are not visible with 'increase contrast'

### ISSUE THREAD

**Issue by csuciu** [mozilla-mobile/firefox-ios#5778]
v20.2(16690)
iPhone XS (13.2.2)

1. Make sure Light theme is enabled in Firefox.
2. Go iOS settings => Accessibility => Display & Text Size and enable the "Increase Contrast" option.
3. Go to iOS settings => Display & Brightness and enable Dark theme.
3. Open Firefox settings page

Result: The separation lines between settings and the right arrows corresponding to each setting are not visible.

![Screen Shot 2019-11-19 at 10 16 41](https://user-images.githubusercontent.com/1866605/69144571-3c588480-0ad4-11ea-8e4a-1ca8875d4d5b.png)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-2201)

**Comment by eyablokov** [comment]
Also, when I open Firefox, elements of the top panel (time, battery, LTE, etc) aren't visible by the same reason - Firefox has white background & the panel has it, too.

**Comment by data-sync-user** [comment]
➤ Asa Dotzler commented:

I’m unable to test this right now but if it’s still present, it’s an accessibility S2.
